CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -1265,7 +1277,7 @@ CFDictionaryRef __CFGetEnvironment() {
         extern char **environ;
         char **envp = environ;
 #elif DEPLOYMENT_TARGET_LINUX
-#ifndef environ
+#if !defined(environ) && !defined(__ANDROID__)
```

Comment:
I’m saying it’s needed, honest! Full code is:

#elif DEPLOYMENT_TARGET_LINUX
#if !defined(environ) && !defined(__ANDROID__)
#define environ __environ
#endif
        char **envp = environ;
#endif